front/app/containers/Admin/projects/project/analysis/InputsList/InputListItem.tsx

```diff
@@ -104,7 +104,7 @@ const InputListItem = memo(({ input, onSelect, selected }: Props) => {
             </Box>
           )}
           {(!title_multiloc || isEmpty(title_multiloc)) && (
-            <Box flex="1">
+            <Box flex="1" width="100%">
```

Comment:
I put `w="100%` here so we'll have a conflict 😂

Reply:
BRING IT ON! :dagger: :grin: